src/main/java/shedar/mods/ic2/nuclearcontrol/crossmod/appeng/BlockNetworkLink.java

```diff
@@ -22,7 +22,7 @@
     public BlockNetworkLink() {
         super(Material.iron);
         this.setBlockName("NetworkLink");
-        this.setBlockHardness(1.0F);
+        this.setHardness(1.0F);
```

Comment:
It was on my side it was nessesary to change it because else MC would not start.